### ISSUE TITLE
Not able to save some substances

### ISSUE THREAD

**Issue by Millali** [HPDCollaborative/builder-ui-beta-reporting#50]
**Describe the bug**
I am not able to add some substances. I am able to add many but I am having trouble with a couple. The ones that I am not able to add have something in common: Their ID from pharos is "Not Registered" 

**To Reproduce**
1. Go to any record, you can use the one below if you want: 
https://staging.hpd-collaborative.org/actions/6419/record
2. Go to substances and click "Add a substance"
3. Search for either of these substances and select to continue
- Kraft paper
- GLASS FILAMENTS 
4. Enter the rest of substance information, make sure to select a material and click save substance

**Expected behavior**
Nothing happens once "Save substance" is clicked. 

**Screenshots**

**Desktop (please complete the following information):**
 - Windows
 - Chrome

**Comment by 19peaches** [comment]
The Kraft Paper is throwing a green screen score error. But I'm not sure where that's occurring because it doesn't happen locally on my machine.

I still need to check the other one.

**Comment by 19peaches** [comment]
Yeah the other is the same thing ... it's not the pharos id, everything that comes up in the pharos lookup is going to have an id ... it has to. This is a green screen error thrown when the request is made to the controller ... I'll dig into this and see what I can find.

**Comment by 19peaches** [comment]
Can you get on slack please

**Comment by 19peaches** [comment]
@Millali please log into Slack
